src/libs/ui/docsetsdialog.cpp

```diff
@@ -95,7 +97,10 @@ DocsetsDialog::DocsetsDialog(Core::Application *app, QWidget *parent)
 #endif
 
     ui->statusLabel->clear(); // Clear text shown in the designer mode.
-    ui->readOnlyLabel->setVisible(m_isStorageReadOnly);
+    ui->storageStatusLabel->setVisible(m_isStorageReadOnly);
+    ui->storageStatusLabel->setText(fi.exists() ?
+                                    QStringLiteral("<b>Docset storage is read only.</b>") :
```

Comment:
FYI, user visible strings should be wrapped into [`tr()`](https://doc.qt.io/qt-5/qobject.html#tr), so they would be translatable.